src/modules/modal/modalpopup.js

```diff
@@ -0,0 +1,102 @@
+import getComments from './getComments.js';
+import postComments from './postcomments.js';
+
+const Modal = async (dataname, dataimagemedium, dataid, datasummary) => {
+    const Modal = document.querySelector('.modal');
+    const modalPopUp = document.createElement('div');
+    modalPopUp.classList.add('modal-popUp');
+
+    const imageSummary = document.createElement('div');
+    imageSummary.classList.add('image-summary'); 
+    const imagePopUp = document.createElement('img');
+    imagePopUp.classList.add('image-popup');
+    imagePopUp.src = dataimagemedium;
+
+    const dataSummary = document.createElement('div');
+    dataSummary.classList.add('data-summary');
+    dataSummary.innerHTML = datasummary;
+    imageSummary.append(imagePopUp);
```

Comment:
# Important 🔴 🔴 🔴 
**Please don't forget that this branch of the pop-up has to print the title, image, and SUMMARY of the series.
In you next branch you must show: the number of comments, the comments, and a functional form to submit the comments and post on the API.**